packages/react/tabs/src/Tabs.stories.tsx

```diff
@@ -205,6 +205,7 @@ const listClass = css({
 });
 
 const RECOMMENDED_CSS__TABS__TRIGGER = {
+  all: 'unset' as any,
```

Comment:
The `triggerClass` threw a huuuuuge TS error if I didn't cast this 🤔

![image](https://user-images.githubusercontent.com/175330/142258092-a054610a-cf62-43fe-932c-f14c81ba2a79.png)

![image](https://user-images.githubusercontent.com/175330/142258125-0cbce384-fd7e-481d-abf2-ea3f019db31c.png)

Reply:
weird…